acceptance-tests/helpers/apps/push.go

```diff
@@ -77,6 +80,34 @@ func WithName(name string) Option {
 		a.Name = name
 	}
 }
+func (a *App) CleanFileFromAppDir(filename string) {
+	err := os.Remove(fmt.Sprintf("%s/%s", a.dir, filename))
+	if err != nil {
+		panic(err)
+	}
+
+}
+func WithYAMLFile(filename string, contents map[string]interface{}) Option {
+	return func(a *App) {
+		//convert to yaml parsable by hil
+		for k, v := range contents {
+			valueBytes, err := json.Marshal(v)
+			if err != nil {
+				panic(err)
+			}
+			contents[k] = string(valueBytes)
+		}
+		bytes, err := yaml.Marshal(contents)
+		if err != nil {
+			panic(err)
+		}
+		err = os.WriteFile(fmt.Sprintf("%s/%s", a.dir, filename), bytes, 0666)
```

Comment:
issue: it's better to use file path functions rather than string interpolation because it's easier to understand and leads to fewer bugs. Instead of:
```go
err = os.WriteFile(fmt.Sprintf("%s/%s", a.dir, filename), bytes, 0666)
if err != nil {
  panic(err)
}
```
do:
```go
Expect(os.WriteFile(filepath.Join(a.dir, filename), bytes, 0666)).To(Succeed())
```